src/_guides/bump-sh-tutorials/generate-sdks-with-speakeasy.md

```diff
@@ -219,9 +219,7 @@ Amazing! The TypeScript SDK is now integrated into the API documentation, and th
 
 ## Step 6: Applying Code Sample Overlays on Bump.sh Deployments
 
-To get this deployed you can do the same thing you just did, run the overlay then publish the resulting document. This will work on any continuous integration solution, just deploy the temporary created document instead of the original.
-
-In GitHub Actions, it will work like this:
+Get the API deployed with overlays is very similar, only instead of creating the overlay ourselves we can let the `bump deploy` command take care of that. Whatever continuous integration solution is being used will be fairly similar, but in GitHub Actions it look like this:
```

Comment:
```suggestion
Getting the API deployed with overlays is very similar, only instead of creating the overlay ourselves we can let the `bump deploy` command take care of that. Whatever continuous integration solution is being used will be fairly similar, but in GitHub Actions it look like this:
```